circle.yml

```diff
@@ -8,12 +8,10 @@ dependencies:
   override:
     - npm install
     - npm install coveralls
-    - npm install bithound
 
 test:
   override:
     - npm test
     - COVERALLS_SERVICE_NAME="circleci" COVERALLS_SERVICE_JOB_ID="$CIRCLE_BUILD_NUM" ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info
-    - ./node_modules/.bin/bithound check git@github.com:fintechdev/x2-connector.git
```

Comment:
Non of the SPA's has it (Neither the services I think). We are not running bithound in each commit anymore. Just for PR's.